reference/redis.html.md

```diff
@@ -121,3 +121,5 @@ Note that items marked with an explicit TTL will expire accurately, regardless o
 |Price per month, per region|$0|$10|$90|
 
 More plans may be added in the future. [Get in touch](https://community.fly.io) on our community forum with feedback and suggestions for improvements.
+
+** Redis is a registered trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by Fly is for referential purposes only and does not indicate any sponsorship, endorsement or affiliation between Redis and Fly.
```

Comment:
```suggestion
** Redis is a registered trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by Fly.io, Inc. is for referential purposes only and does not indicate any sponsorship, endorsement or affiliation between Redis and Fly.io, Inc.
```

I'm suggesting this change but I don't know if we gain anything from the "Inc." Will have to check internally.